### PR TITLE
Navbar restyle

### PR DESCRIPTION
**Pull request by Shruti-Apte**
## Notion card
https://www.notion.so/saeloun/Change-the-navigation-on-Home-page-0af62fe9d5ac460fb9e1152618f3b21a
## Summary

- Navbar updated according to new designs
## Preview

Screenshot 2022-07-27 at 6 32 42 PM

## Type of change

- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code

### CODE REVIEW

**Comment by onkar-saeloun** [comment]
@Shruti-Apte checks are failing

File: app/javascript/src/components/Navbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+              : "flex py-3 px-6 items-center hover:bg-miru-gray-100"
+          }
+        >
+          {option.logo} {option.label}
+        </NavLink>
+      </li>
+    ));
+
+  const getAdminOption = () =>
+    navAdminOptions.map((option) => (
```
Comment (onkar-saeloun) [inline app/javascript/src/components/Navbar/index.tsx:109]: same as above

File: app/javascript/src/components/Navbar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    <div className="shadow-2xl w-1/6 h-full flex flex-col justify-between ">
+      <div>
+        <div className="h-20 bg-miru-gray-100 flex items-center justify-center">
+          <img src={miruLogo} />
+        </div>
+        <ul className="mt-8">
+          {isAdminUser ? getAdminOption() : getEmployeeOptions()}
+        </ul>
+      </div>
+
```
Comment (onkar-saeloun) [inline app/javascript/src/components/Navbar/index.tsx:134]: no extra space

File: app/javascript/src/components/Navbar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+            </li>
+          </a>
+          <a data-method="delete" href="/users/sign_out" rel="nofollow">
+            <li className="flex py-3 px-6 items-center hover:bg-miru-gray-100">
+              <SignOut size={26} className="mr-4" /> Logout
+            </li>
+          </a>
+        </ul>
+        <div className="mr-2 w-full h-16 p-4 bg-miru-gray-100 flex items-center">
+          <img src={avatar} className="mr-2" />
```
Comment (onkar-saeloun) [inline app/javascript/src/components/Navbar/index.tsx:159]: while using  tag, it is good practice to use alt attribute

**Review by apoorv1316 — CHANGES_REQUESTED** [review]
@Shruti-Apte Can you remove the dashboard tab? We don't have any route for the dashboard tag. It throws an error when you click on the dashboard. Also, the navbar is open all the time. Do we want it open all the time or a button to open and close the navbar @supriya3105

**Review by onkar-saeloun — APPROVED** [review]

**Review by AkashKale — COMMENTED** [review]
Discussed following changes with Shruti :

- Font weight for selected section should be 800 (Extrabold)
- Spacing between bottommost section (logout) and profile section should be 24px (1.5rem)
- Update miru logo and profile image to 2x because currently it is looking blurred
- Update hover effect
- Update text : 'Setting' to 'Settings'

**Review by apoorv1316 — APPROVED** [review]

**Pull request merged**